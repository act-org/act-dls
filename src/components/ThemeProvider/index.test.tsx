/**
 * Copyright (c) ACT, Inc. and its affiliates.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * @prettier
 */

import * as React from 'react';

import render from '../../helpers/test/render';

import { ThemeProvider } from '.';

describe('ThemeProviderPrimary', () => {
  it('matches the snapshot', () => {
    const { container } = render(
      <ThemeProvider theme="act">
        <p>This is a test</p>
      </ThemeProvider>,
    );

    expect(container).toMatchSnapshot();
  });
});
