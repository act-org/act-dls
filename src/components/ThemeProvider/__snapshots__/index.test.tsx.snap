// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ThemeProviderPrimary matches the snapshot 1`] = `
<div>
  <p>
    This is a test
  </p>
</div>
`;
